src/NATS.Client.JetStream/Internal/NatsJSJsonSerializer.cs

```diff
@@ -335,3 +337,30 @@ public override TimeSpan Read(ref Utf8JsonReader reader, Type typeToConvert, Jso
     public override void Write(Utf8JsonWriter writer, TimeSpan value, JsonSerializerOptions options) =>
         writer.WriteNumberValue((long)(value.TotalMilliseconds * 1_000_000L));
 }
+
+internal class NatsJSJsonNullableNanosecondsConverter : JsonConverter<TimeSpan?>
```

Comment:
Have created this to support a nullable `TimeSpan?` when used for the `PauseRemaining` in the `ConsumerInfo`.

Reply:
Could you add a test for this in https://github.com/nats-io/nats.net.v2/blob/b47095a7fb9d020a4d83d4bba54ea51b635e56bd/tests/NATS.Client.JetStream.Tests/TimeSpanJsonTests.cs